Packs/CommonTypes/IndicatorTypes/reputation-ipv6.json

```diff
@@ -7,7 +7,7 @@
     "commitMessage": "",
     "shouldPublish": false,
     "shouldCommit": false,
-    "regex": "(?:[a-fA-F0-9]{1,4}:{1,2}){3,7}[a-fA-F0-9]{1,4}",
+    "regex": "(?i)(?P<normal>(?:[0-9a-f]{1,4}:){7,7}[0-9a-f]{1,4})|(?P<local>fe80:(?::[0-9a-f]{0,4}){0,4}%[0-9a-zA-Z]+)|(?P<ipv4_embedded>::(?:ffff(?::0{1,4}){0,1}:){0,1}(?:(?:25[0-5]|(?:2[0-4]|1{0,1}[0-9]){0,1}[0-9])[.]){3,3}(?:25[0-5]|(?:2[0-4]|1{0,1}[0-9]){0,1}[0-9])|(?:[0-9a-f]{1,4}:){1,4}:(?:(?:25[0-5]|(?:2[0-4]|1{0,1}[0-9]){0,1}[0-9])[.]){3,3}(?:25[0-5]|(?:2[0-4]|1{0,1}[0-9]){0,1}[0-9]))|(?P<collapsed>:(?:(?::[0-9a-f]{1,4}){1,7})|[0-9a-f]{1,4}:(?:(?::[0-9a-f]{1,4}){1,6})|(?:[0-9a-f]{1,4}:){1,2}(?::[0-9a-f]{1,4}){1,5}|(?:[0-9a-f]{1,4}:){1,3}(?::[0-9a-f]{1,4}){1,4}|(?:[0-9a-f]{1,4}:){1,4}(?::[0-9a-f]{1,4}){1,3}|(?:[0-9a-f]{1,4}:){1,5}(?::[0-9a-f]{1,4}){1,2}|(?:[0-9a-f]{1,4}:){1,6}:[0-9a-f]{1,4}|(?:[0-9a-f]{1,4}:){1,7}:)",
```

Comment:
Regarding the regex it might be better the @MLainer1 will check this as part of TIM ownership.

Reply:
In the regex department, Dror is the best :)